src/server/game/AuctionHouseBot/AuctionHouseBotSeller.h

```diff
@@ -39,6 +39,7 @@ struct SellerItemClassInfo
     uint32 MissItems;
     uint32 Quantity;
     uint32 PriceRatio;
+    uint32 RandomStackRatio;
```

Comment:
Missing initialization in https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/AuctionHouseBot/AuctionHouseBotSeller.h#L36